src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/ProfessionalExternalUserController.java

```diff
@@ -134,11 +134,7 @@ public ResponseEntity findUsersByOrganisation(@ApiParam(hidden = true) @OrgId St
                     @Authorization(value = "Authorization")
             }
     )
-    @ApiParam(
-            name = "email",
-            type = "string",
-            value = "The Email Address of the User to be retrieved"
-    )
+
```

Comment:
Check with BA,  if we can remove Email address description from swagger

Reply:
yes we can it is no use.

Reply:
done